python/lsst/meas/base/catalogCalculation.py

```diff
@@ -144,26 +146,29 @@ def initializePlugins(self):
         pluginType = namedtuple('pluginType', 'single multi')
         self.executionDict = {}
         # Read the properties for each plugin. Allocate a dictionary entry for each run level. Verify that
-        # the plugins are above the minimum run level for an afterburner plugin. For each run level, the
-        # plugins are sorted into either single record, or multi record groups to later be run appropriately
+        # the plugins are above the minimum run level for an catalogCalculation plugin. For each run level,
+        # the plugins are sorted into either single record, or multi record groups to later be run
+        # appropriately
         for executionOrder, name, config, PluginClass in self.config.plugins.apply():
             if executionOrder not in self.executionDict:
                 self.executionDict[executionOrder] = pluginType(single=[], multi=[])
-            if PluginClass.getExecutionOrder() >= BasePlugin.DEFAULT_AFTERBURNER:
+            if PluginClass.getExecutionOrder() >= BasePlugin.DEFAULT_CATALOGCALCULATION:
                 plug = PluginClass(config, name, self.schema, metadata=self.plugMetadata)
                 self.plugins[name] = plug
                 if plug.plugType == 'single':
                     self.executionDict[executionOrder].single.append(plug)
                 elif plug.plugType == 'multi':
                     self.executionDict[executionOrder].multi.append(plug)
             else:
-                raise ValueError("{} has an execution order less than the minimum for an afterburner plugin."
-                                 "Value {} : Minimum {}".format(PluginClass, PluginClass.getExecutionOrder(),
-                                                                BasePlugin.DEFAULT_AFTERBURNER))
+                raise ValueError("{} has an execution order less than the minimum for an catalogCalculation "
+                                 "plugin. Value {} : Minimum {}".format(PluginClass,
+                                                                        PluginClass.getExecutionOrder(),
+                                                                        BasePlugin.DEFAULT_CATALOGCALCULATION)
+                                )
```

Comment:
because one more character takes the line past its maximum 110 character limit, so it flows to the next line indented to the same level as the opening parentheses. You are right about there not being anything special about parentheses being on their own lines.